core/src/main/java/resource/fld/core-base.fld

```diff
@@ -9930,6 +9930,38 @@ You can find a list of all available timezones in the Lucee administrator (Setti
 		</return>
 	</function>
 
+	<!-- queryRenameColumn -->
+	<function>
+		<name>queryRenameColumn</name>
+		<class>lucee.runtime.functions.query.QueryRenameColumn</class>
+		<keywords>query</keywords>
+		<member-name>renameColumn</member-name>
+		<description>Rename a column</description>
+		<introduced>6.0.0</introduced>
+		<argument>
+			<name>query</name>
+			<type>query</type>
+			<required>Yes</required>
+			<description>Name of a query object.</description>
```

Comment:
it is not the name of the query,it is the query itself. it could be for example queryRenameColumn(queryExecute(...),...)